### PR TITLE
fix: speed up slow flow board using static methods

### LINKED ISSUE #7329: bug: flow board is slow
### Describe the bug

https://community.open-emr.org/t/openemr-7-0-2-extremely-slow/22313/46

### To Reproduce

1. Add a '....'

### Client configuration

- Browser:
- OpenEMR version:
- Operating system:

### PR DESCRIPTION
**Pull request by stephenwaite**
Fixes #7329

### CODE REVIEW

File: library/patient_tracker.inc.php [hunk 1]
```diff
@@ -32,9 +32,12 @@
 use OpenEMR\Services\AppointmentService;
 use OpenEMR\Services\PatientTrackerService;
 
+$apppointmentService = new AppointmentService();
+$patientTrackerService = new PatientTrackerService();
```
Comment (bradymiller) [inline library/patient_tracker.inc.php:36]: Are their specific queries/code that are slow in these classes during their initiation?

Comment (stephenwaite) [reply library/patient_tracker.inc.php:36]: Not seeing slow instantiation, bogs down from hundreds of unnecessary new objects.

File: src/Services/PatientTrackerService.php [hunk 2]
```diff
@@ -167,7 +167,7 @@ public function is_tracker_encounter_exist($apptdate, $appttime, $pid, $eid)
      * @param string $enc_id
      * @return bool|int
      */
-    public function manage_tracker_status($apptdate, $appttime, $eid, $pid, $user, $status = '', $room = '', $enc_id = '')
+    public static function manage_tracker_status($apptdate, $appttime, $eid, $pid, $user, $status = '', $room = '', $enc_id = '')
```
Comment (bradymiller) [inline src/Services/PatientTrackerService.php:170]: There is a $this used in line 273, which will likely break. Not sure how to rework that to work for a static call. @adunsulag thoughts?

Comment (adunsulag) [reply src/Services/PatientTrackerService.php:170]: We'd need to use a singleton instance (have a static $instance property that is initialized once), or skip refactoring this method if its not used in the flowboard.

**Review by bradymiller — APPROVED** [review]

**Comment by bradymiller** [comment]
tenor99

**Pull request merged**